### ISSUE TITLE
Excluding amendments

### ISSUE THREAD

**Issue by gregjasonroberts** [joeyism/py-edgar#19]
What would you suggest for applying a parameter for excluding amendments?  I know in the sec_edgar_downloader library there's a parameter that can be set: include_amends=False.   Do you have a parameter in your methods or can direct me how I can apply that to the code below?
Thanks again!

I'm using the following code to pull the documents:

```python
from edgar import Company
from edgar import TXTML

company = Company("BOSTON SCIENTIFIC CORP", "0000885725")
tree = company.get_all_filings(filing_type = "10-K")
docs = Company.get_documents(tree, no_of_documents=5)

text=TXTML.parse_full_10K(docs[3])
text
```

**Comment by joeyism** [comment]
I'm not familiar with what amendments. Can you explain that further? Is that the `10-K/A`?

**Comment by gregjasonroberts** [comment]
That's correct.  When pulling the 10K documents in this method, 10-K/A documents are included.  In the example above, the 4th document scraped out of the 5 documents called contained this and I'd like to skip those types of filings.
